apps/blockchain/lib/blockchain/interface/account_interface.ex

```diff
@@ -382,6 +382,7 @@ defimpl EVM.Interface.AccountInterface, for: Blockchain.Interface.AccountInterfa
           EVM.MachineCode.t(),
           integer(),
           Block.Header.t(),
+          EVM.address(),
```

Comment:
Since you're passing `nil` in the doc test above, I think dialyzer will immediately say this is incorrect (even if we're suppressing the warning) because this argument should be `EVM.address() | nil` . Should we add the `nil` to the typespec or does it make sense to never pass `nil` into this?

Reply:
no, we can't.  because created account address is only passed if a contract is being created with opcode ( from evm). Contract creation transactions pass nil